extensions/pyRevitTools.extension/pyRevit.tab/Selection.panel/memo.stack3/Copy State.pushbutton/script.py

```diff
@@ -319,33 +316,43 @@ def set_tansform_matrix(selvp, selview):
                                       file_ext='pym',
                                       add_cmd_name=False)
 
-    av = revit.activeview
+    selected_elements = revit.get_selection().elements
+    selected_views = copy_paste_state_utils.get_views(selected_elements)
```

Comment:
_here and below_ get all selected views (including views of viewports or an active view)